stl/inc/atomic

```diff
@@ -351,6 +373,32 @@ struct _Atomic_storage {
         return _Result;
     }
 
+#if _HAS_CXX20
+
+    void wait(_Ty _Expected, const memory_order _Order = memory_order_seq_cst) const noexcept {

```

Comment:
http://wg21.link/p1135 has no const, actually there should be no const for _Order too

Reply:
Top-level `const` on parameters isn't user-observable since it's not part of the function signature. We typically add top-level `const` to parameters that we don't intend to change within the function body to avoid errors.

Reply:
I had to add more `const_cast` for Windows API calls. But these API functions don't say they change parameter, so looks like consts are valid.